docs/pages/Configuration.md

```diff
@@ -449,11 +449,20 @@ Unless otherwise noted in the table below, the UI5 Tooling modules are backward
 
 Version | UI5 CLI Release
 --- | ---
+**2.1** | v2.1.0+
 **2.0** | v2.0.0+
 **1.1** | v1.13.0+
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 2.1
+
+**Features:**
+
+- Adds support for the ["customConfiguration"](#custom-configuration) configuration to provide custom data. Must be handled by custom tools. Can be consumed when using @ui5/project API.
+
+Version 2.1 projects are supported by [UI5 CLI](https://github.com/SAP/ui5-cli) v.2.1.0 and above.
```

Comment:
What versions are these? Is Version 2.1 the UI5 Tooling version or the specVersion? And how about the UI5 CLI Release v.2.1.0 - is that yet another different version? This is not clear to me.

Reply:
With `Version 2.1` the specVersion is meant.
And UI5 CLI v2.1.0 is the upcoming UI5 CLI release that supports specVersion 2.1.

We should make this a bit more clear. The text is actually taken from the previous version descriptions below, so we should update all of them.

Reply:
Would give consideration to moving the section '## Specification Versions' to the beginning, since it's such a central concept for the configuration of UI5 Tooling. It's mentioned all over the place.